ui/src/lib/translations/en/common.json

```
@@ -28,5 +28,16 @@
   "what_is_your_name": "What is your name?",
   "need_camera_permission": "Camera permission is required to scan codes.",
   "open_app_settings": "Open App Settings",
-  "holochain_connect_error": "Error connecting to Holochain"
+  "holochain_connect_error": "Error connecting to Holochain",
+  "profile_update_error": "Failed to update profile",
+  "copy_success": "Copied to clipboard",
+  "copy_error": "Failed to copy",
+  "create_account_error": "Failed to create account",
+  "share_text_error": "Failed to share code",
```

```suggestion
  "share_text_error": "Failed to share code",
```

Lets either make this `share_code_error`, or change the text to be "Failed to share text"